docs/_articles/ja/apex/apex-debugging.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Debugging Apex
+lang: en
+---
+
+The Salesforce VS Code Extensions pack comes with rich debugging options for all levels of developer experiences. Use this information to figure out which Apex debugger is right for you:
```

Comment:
maybe ...debugging options for every developer, regardless of where you land on the low- to pro-code continuum.